erizo_controller/erizoJS/models/Client.js

```diff
@@ -23,6 +23,7 @@ class Client extends EventEmitter {
     this.ioThreadPool = ioThreadPool;
     this.singlePc = singlePc;
     this.streamPriorityStrategy = Client._getStreamPriorityStrategy(streamPriorityStrategy);
+    this.connectionTargetBw = options.connectionTargetBw || 0;
```

Comment:
I'm not sure I like this third way of adding it. Shouldn't we obtain the value from the streamPriorityStrategy?

```js
this.connectionTargetBw = this.streamPriorityStrategy.connectionTargetBw;
```

Reply:
The value may have been updated, erizoController sends the last version with every addPublisher or addSubscriber. Here we're accounting for the case where this client was not present in this erizoJS